frontend/src/pages/Workflows/actions.js

```diff
@@ -40,6 +40,9 @@ export const WORKFLOW_EDIT = "WORKFLOW_EDIT";
 export const SHOW_WORKFLOW_DETAILS = "SHOW_WORKFLOW_DETAILS";
 export const HIDE_WORKFLOW_DETAILS = "HIDE_WORKFLOW_DETAILS";
 export const CLOSE_WORKFLOWITEM_DETAILS = "CLOSE_WORKFLOWITEM_DETAILS";
+export const DIALOG_CLOSE_WORKFLOWITEM = "DIALOG_CLOSE_WORKFLOWITEM";
+export const HIDE_DIALOG_FOR_CLOSE = "HIDE_DIALOG_FOR_CLOSE";
+export const DIALOG_CLOSE_SUBPROJECT = "DIALOG_CLOSE_SUBPROJECT";
```

Comment:
If you want to split subproject's and workflowitem's closing dialog split everything related to them( component - already done and actions- bit mixed). Additionally actions should be unique, but HIDE_DIALOG_FOR_CLOSE is also defined in overview/actions.
```suggestion
export const SHOW_DIALOG_CLOSE_WORKFLOWITEM = "SHOW_DIALOG_CLOSE_WORKFLOWITEM ";
export const HIDE_DIALOG_FOR_CLOSE_WORKFLOWITEM = "HIDE_DIALOG_FOR_CLOSE_WORKFLOWITEM ";
export const SHOW_DIALOG_CLOSE_SUBPROJECT = "SHOW_DIALOG_CLOSE_SUBPROJECT ";
export const HIDE_DIALOG_FOR_CLOSE_SUBPROJECT = "HIDE_DIALOG_FOR_CLOSE_SUBPROJECT ";
```